MaxText/configs/base.yml

```diff
@@ -16,6 +16,10 @@
 # If there is already a checkpoint under this run, that checkpoint will auto-resume.
 run_name: ""
 
+# Types of models supported.
+# Currently supported models: gpt3
+model_name: "gpt3"
```

Comment:
Current model is "palm" like.

Reply:
But I think we should call it `default` for now -- it is really saying that if you use this model we don't use any overrides.

Reply:
(And if you say anything else we will mix in some overrides for you.)